File: js/1015.js
var input = require('fs').readFileSync('/dev/stdin', 'utf8');
var lines = input.split('\n');
const [x1, y1] = lines[0].split(' ').map(Number);
const [x2, y2] = lines[1].split(' ').map(Number);
const calculo = Math.sqrt(Math.pow((x2 - x1), 2) + Math.pow((y2 - y1), 2));
console.log(calculo.toFixed(4));

File: cs/1015.cs
String linha1 = Console.ReadLine();
String linha2 = Console.ReadLine();
string[] p1 = linha1.Split(' ');
string[] p2 = linha2.Split(' ');
double x1 = double.Parse(p1[0]);
double y1 = double.Parse(p1[1]);
double x2 = double.Parse(p2[0]);
double y2 = double.Parse(p2[1]);
double calculo = Math.Sqrt(Math.Pow((x2 - x1), 2) + Math.Pow((y2 - y1), 2));
Console.WriteLine(calculo.ToString("0.0000"));